lang/fr_FR.UTF-8/LC_MESSAGES/help.po

```diff
@@ -7219,3 +7219,24 @@ msgid "Mode used to retrieve information displayed in Resource Status page."
 "The optimized mode will only be available and functional when using the BBDO 3.0.0 protocol for the collect engine."
 msgstr "Mode utilisé pour récupérer les données affichées dans la page Resource Status."
 "Le mode optimisé pourra être utilisé et fonctionnel que si le moteur de supervision utilise le protocole BBDO 3.0.0"
+
+msgid "Poller configuration actions / Poller management"
+msgstr "Gestion des serveurs de collecte/collecteurs"
+
+msgid "Create and edit pollers"
+msgstr "Créer et modifier des collecteurs"
+
+msgid "Delete pollers"
+msgstr "Supprimer des collecteurs"
+
+msgid "Deploy configuration"
+msgstr "Déployer la configuration"
+
+msgid "Allows user to perform “Add”, “Add (advanced)” and “Duplicate” actions on pollers."
+msgstr "Permet aux utilisateurs de créer des collecteurs (boutons “Ajouter”, Ajouter (avancé)” et “Dupliquer”)"
```

Comment:
use infinitive

Reply:
not needed for consistency with other help tooltip

Reply:
Infinitive is not use on this page